frontend/src/layouts/BaseLayout.vue

```diff
@@ -248,13 +226,11 @@ import BaseTooltip from 'src/components/BaseTooltip.vue';
 import ConnectWallet from 'src/components/ConnectWallet.vue';
 import useSettingsStore from 'src/store/settings';
 import useWalletStore from 'src/store/wallet';
-import AddressSettings from './AddressSettings.vue';
 import HeaderLinks from './HeaderLinks.vue';
-import NetworkDropdown from './NetworkDropdown.vue';
 
 export default defineComponent({
   name: 'BaseLayout',
-  components: { AddressSettings, ArgentWarningModal, BaseButton, BaseTooltip, ConnectWallet, HeaderLinks, NetworkDropdown }, // prettier-ignore
+  components: { ArgentWarningModal, BaseButton, BaseTooltip, ConnectWallet, HeaderLinks }, // prettier-ignore
```

Comment:
_(arbitrary comment location to start a thread)_

1. Do we have any control over the style of this dropdown? It would be nice to have a themed dropdown here that matches the app style / previous style and shows the logos for each chain
2. The "Explore" link for smart contracts takes me to the FAQ. Is this expected? My guess was that it was going to link me to the Umbra contract on Etherscan for the currently selected chain

<img width="199" alt="image" src="https://user-images.githubusercontent.com/17163988/197589821-ba6a881a-b6ef-4c5f-bc40-acc3b72b92f1.png">